scripts/integration-test.sh

```diff
@@ -1,18 +1,14 @@
 #!/bin/bash
-INTEGRATION_TEST=$1
-SRC_PATH=$2
-GATSBY_PATH="${TRAVIS_BUILD_DIR:-../../}" # set to third arg if defined, otherwise use ../../
+SRC_PATH=$1
+CUSTOM_COMMAND="${2:-test}"
+GATSBY_PATH="/home/circleci/project"
```

Comment:
`CIRCLE_WORKING_DIRECTORY` annoyingly includes a `~`, e.g. `~/project`

In the container, Node resolves this (for some reason) to

```js
const path = require('path');

path.resolve(`~/project`); // /home/circleci/project/integration-tests/path-prefix/~/project
```

which is a non-existant path.

Reply:
Also I tried relative directory here, but that seemed to fail too. `../../` should be the ticket but didn't quite work consistently. It's possible changed to `gatsby-dev-cli` will be necessary to fully resolve this behavior.